.gitignore

```diff
@@ -4,8 +4,7 @@
 */*/__pycache__/
 */__pycache__/
 docs/build/
-venv/
-env/
+.venv/
```

Comment:
Let's also ignore ./env and ./venv or are you specifially using the hidden folder name .venv for a virtual environment?

Reply:
yes, that is the recommendation in the official documentation and that is also what I took as error message instruction and what is in the readme

Reply:
Which official documentation or readme are you referring to?

Reply:
https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/